script/scriptgen.py

```diff
@@ -486,7 +486,7 @@ def gen_if_not_customized(self, folder, fname):
     def gen_repo(self, repodir, gen, f):
         if "$build" in gen:
             self.p(
-                """build=$(grep -o -E '(Build|Snapshot)[^-]*' __envsub/files_iso.lst | grep -o -E '[0-9]\.?[0-9]+(\.[0-9]+)*' | head -n 1)""",
+                """build=$(grep -o -E '(Build|Snapshot)[^-]*' __envsub/files_iso.lst | grep -o -E '[0-9]\\.?[0-9]+(\\.[0-9]+)*' | head -n 1)""",
```

Comment:
why not use the suggested raw string? just add `r` in the beginning of the string! wouldnt that work? I think it would simplify the expression. Adding slashes it is harder to read sometimes. wdyt?

Reply:
sure, i don't mind much. It doesn't seem to break anything else, so done